This is `make` clone written using sbt (https://github.com/sbt/sbt/tree/1.10.x/sbt-app/src/sbt-test/nio/make-clone). Likely it needs to be adjusted to the musl toolchain that's now on the PATH?
 
```bash
[info] Running nio/make-clone
[info] [info] welcome to sbt 1.10.4-SNAPSHOT (Eclipse Adoptium Java 21.0.4)
[info] [info] loading settings for project global-plugins from metals.sbt ...
[info] [info] loading global plugins from /tmp/sbt_e3f41de7/global/plugins
[info] [info] loading project definition from /tmp/sbt_e3f41de7/project
[info] [info] compiling 1 Scala source to /tmp/sbt_e3f41de7/project/target/scala-2.12/sbt-1.0/classes ...
[info] [info] done compiling
[info] [success] Generated .bloop/sbt_e3f41de7-build.json
[info] [success] Total time: 0 s, completed Oct 23, 2024, 8:29:06 AM
[info] [info] loading settings for project sbt_e3f41de7 from build.sbt,tests.sbt ...
[info] [warn] Discarding 4 session settings.  Use 'session save' to persist session settings.
[info] [info] set current project to sbt_e3f41de7 (in build file:/tmp/sbt_e3f41de7/)
[info] [warn] there are 2 keys that are not used by any other settings/tasks:
[info] [warn]  
[info] [warn] * sbt_e3f41de7 / compileLib / target
[info] [warn]   +- /tmp/sbt_e3f41de7/build.sbt:12
[info] [warn] * sbt_e3f41de7 / linkLib / target
[info] [warn]   +- /tmp/sbt_e3f41de7/build.sbt:46
[info] [warn]  
[info] [warn] note: a setting might still be used by a command; to exclude a key from this `lintUnused` check
[info] [warn] either append it to `Global / excludeLintKeys` or call .withRank(KeyRanks.Invisible) on the key
[info] [info] Compiling /tmp/sbt_e3f41de7/src/lib/lib.c to /tmp/sbt_e3f41de7/target/streams/_global/compileLib/_global/streams/lib.o
[info] [info] Linking /tmp/sbt_e3f41de7/target/streams/_global/linkLib/_global/streams/libfoo.so
[info] [info] Building executable /tmp/sbt_e3f41de7/out/main/main.out
[info] [info] Running /tmp/sbt_e3f41de7/out/main/main.out with arguments: '1'
[info] [error] java.io.IOException: Cannot run program "/tmp/sbt_e3f41de7/out/main/main.out": error=2, No such file or directory
[info] [error] 	at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1170)
[info] [error] 	at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1089)
[info] [error] 	at RunBinary$.apply(RunBinary.scala:10)
[info] [error] 	at $585ecb8736b3c86dd54c$.$anonfun$$sbtdef$5(/tmp/sbt_e3f41de7/build.sbt:108)
[info] [error] 	at sbt.util.LoggerContext$LoggerContextImpl$Log.log(LoggerContext.scala:124)
[info] [error] 	at sbt.internal.util.ManagedLogger.log(ManagedLogger.scala:42)
[info] [error] 	at sbt.util.Logger.info(Logger.scala:27)
[info] [error] 	at $585ecb8736b3c86dd54c$.$anonfun$$sbtdef$3(/tmp/sbt_e3f41de7/build.sbt:108)
[info] [error] 	at $585ecb8736b3c86dd54c$.$anonfun$$sbtdef$3$adapted(/tmp/sbt_e3f41de7/build.sbt:99)
....
[info] [error] Caused by: java.io.IOException: error=2, No such file or directory
[info] [error] 	at java.base/java.lang.ProcessImpl.forkAndExec(Native Method)
[info] [error] 	at java.base/java.lang.ProcessImpl.<init>(ProcessImpl.java:295)
[info] [error] 	at java.base/java.lang.ProcessImpl.start(ProcessImpl.java:225)
[info] [error] 	at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1126)
[info] [error] 	at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1089)
[info] [error] 	at RunBinary$.apply(RunBinary.scala:10)
[info] [error] 	at $585ecb8736b3c86dd54c$.$anonfun$$sbtdef$5(/tmp/sbt_e3f41de7/build.sbt:108)
....
[info] [error] (executeMain) java.io.IOException: Cannot run program "/tmp/sbt_e3f41de7/out/main/main.out": error=2, No such file or directory
[info] [error] Total time: 0 s, completed Oct 23, 2024, 8:29:08 AM
Error:  x nio/make-clone 
Error:   Cause of test exception: {line 1}  Command failed: executeMain failed
```

It's probably overspecifying as an sbt test, so I'm okay with removing the test.

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -83,6 +83,7 @@ jobs:
         ref: 1.10.x
         path: zinc
     - uses: graalvm/setup-graalvm@v1
+      if: ${{ matrix.jobtype >= 7 && matrix.jobtype <= 9 }}
       with:
         java-version: '23'
         native-image-musl: 'true'

--- a/build.sbt
+++ b/build.sbt
@@ -1198,7 +1198,7 @@ lazy val sbtClientProj = (project in file("client"))
       s"-H:Name=${target.value / "bin" / "sbtn"}",
     ) ++ (if (isLinux && isArmArchitecture)
             Seq("-H:PageSize=65536") // Make sure binary runs on kernels with page size set to 4k, 16 and 64k
-          else Nil) ++ (if (isLinux) Seq("--static", "--libc=musl") else Nil),
+          else Nil) ++ (if (isLinux && !isArmArchitecture) Seq("--static", "--libc=musl") else Nil),
     buildThinClient := {
       val isFish = Def.spaceDelimited("").parsed.headOption.fold(false)(_ == "--fish")
       val ext = if (isWin) ".bat" else if (isFish) ".fish" else ".sh"

